janusgraph-all/src/main/java/org/janusgraph/graphdb/tinkerpop/plugin/JanusGraphGremlinPlugin.java

```diff
@@ -14,61 +14,236 @@
+        CLASS_IMPORTS.add(RelationType.class);
+        CLASS_IMPORTS.add(Transaction.class);
+        CLASS_IMPORTS.add(TransactionBuilder.class);
+        CLASS_IMPORTS.add(VertexLabel.class);
+        CLASS_IMPORTS.add(VertexList.class);
+
+        CLASS_IMPORTS.add(AttributeSerializer.class);
+        CLASS_IMPORTS.add(Cmp.class);
+        CLASS_IMPORTS.add(Contain.class);
+        CLASS_IMPORTS.add(Geo.class);
+        CLASS_IMPORTS.add(Geoshape.class);
+        CLASS_IMPORTS.add(GeoshapeHelper.class);
+        CLASS_IMPORTS.add(JtsGeoshapeHelper.class);
+        CLASS_IMPORTS.add(Text.class);
+
+        CLASS_IMPORTS.add(ConsistencyModifier.class);
+        CLASS_IMPORTS.add(DefaultSchemaMaker.class);
+        CLASS_IMPORTS.add(EdgeLabelMaker.class);
+        CLASS_IMPORTS.add(Index.class);
+        CLASS_IMPORTS.add(JanusGraphConfiguration.class);
+        CLASS_IMPORTS.add(JanusGraphIndex.class);
+        CLASS_IMPORTS.add(JanusGraphManagement.class);
+        CLASS_IMPORTS.add(JanusGraphSchemaElement.class);
+        CLASS_IMPORTS.add(JanusGraphSchemaType.class);
+        CLASS_IMPORTS.add(JobStatus.class);
+        CLASS_IMPORTS.add(Mapping.class);
+        CLASS_IMPORTS.add(Parameter.class);
+        CLASS_IMPORTS.add(PropertyKeyMaker.class);
+        CLASS_IMPORTS.add(RelationTypeIndex.class);
+        CLASS_IMPORTS.add(RelationTypeMaker.class);
+        CLASS_IMPORTS.add(SchemaAction.class);
+        CLASS_IMPORTS.add(SchemaInspector.class);
+        CLASS_IMPORTS.add(SchemaManager.class);
+        CLASS_IMPORTS.add(SchemaStatus.class);
+        CLASS_IMPORTS.add(VertexLabelMaker.class);
+
+        CLASS_IMPORTS.add(GraphOfTheGodsFactory.class);
+        CLASS_IMPORTS.add(JanusGraphIoRegistry.class);
+        CLASS_IMPORTS.add(ConfigurationManagementGraph.class);
+
+        CLASS_IMPORTS.add(Instant.class);
+        CLASS_IMPORTS.add(Clock.class);
+        CLASS_IMPORTS.add(DayOfWeek.class);
+        CLASS_IMPORTS.add(Duration.class);
+        CLASS_IMPORTS.add(LocalDate.class);
+        CLASS_IMPORTS.add(LocalTime.class);
+        CLASS_IMPORTS.add(LocalDateTime.class);
+        CLASS_IMPORTS.add(Month.class);
+        CLASS_IMPORTS.add(MonthDay.class);
+        CLASS_IMPORTS.add(OffsetDateTime.class);
+        CLASS_IMPORTS.add(OffsetTime.class);
+        CLASS_IMPORTS.add(Period.class);
+        CLASS_IMPORTS.add(Year.class);
+        CLASS_IMPORTS.add(YearMonth.class);
+        CLASS_IMPORTS.add(ZonedDateTime.class);
+        CLASS_IMPORTS.add(ZoneId.class);
+        CLASS_IMPORTS.add(ZoneOffset.class);
+
+        ///////////
+        // ENUMS //
+        ///////////
+
+        Collections.addAll(ENUM_IMPORTS, Geo.values());
+        Collections.addAll(ENUM_IMPORTS, Text.values());
+        Collections.addAll(ENUM_IMPORTS, Multiplicity.values());
+        Collections.addAll(ENUM_IMPORTS, Cardinality.values());
+        Collections.addAll(ENUM_IMPORTS, ChronoUnit.values());
+
+        ///////////
```

Comment:
I'm not suggesting that we need comment-style boxes around headings (I'm ambivalent), but if we are going to do this, then this comment block should be updated to be a complete box.